external_trigger: false

segmenting_in_3d: true

number_frames_between_clustering: 10

min_global_motion: 100

use_rm_predictor: false
rm_min_dist: 5

use_rm3d_predictor: true
rm3d_min_dist: 0.04

use_fm_predictor: false
fm_num_hypo: 150000
fm_num_trials_per_hypo: 20

use_sd_predictor: true
sd_min_dist: 0.15

use_ld_predictor: true
ld_min_dist: 0.2
ld_max_dist: 0.5

use_t_predictor: false

use_csegm_predictor: false

use_csimil_predictor: false
